src/etc/inc/system.inc

```diff
@@ -1059,6 +1059,22 @@ EOD;
 		$syslogd_extra = "-f {$g['varetc_path']}/syslog.conf {$sourceip}";
 	}
 
+	$log_sockets = array();
+	$log_sockets[] = "{$g['dhcpd_chroot_path']}/var/run/log";
+
+	if (isset($config['installedpackages']['package'])) {
+		foreach ($config['installedpackages']['package'] as $package) {
+			if (isset($package['logging']['logsocket']) && '' != $package['logging']['logsocket'] &&
+			    is_dir(pathinfo($package['logging']['logsocket'], PATHINFO_DIRNAME)) && !in_array($package['logging']['logsocket'], $log_sockets)) {
```

Comment:
Please break this if sentence in more lines to avoid introducing more long lines.

Also, you can replace:
```
pathinfo(..., PATHINFO_DIRNAME)
```
by
```
dirname(...)
```